### ISSUE TITLE
Validation is not raising error on a subscription with multiple root fields within inline fragment 

### ISSUE THREAD

**Issue by Juliano-Prado** [graphql-java/graphql-java#3425]
**Describe the bug**
GraphqlJava validation SubscriptionUniqueRootField is not returning error of multiple root fields when there are multiple fields inside root [inline fragment ](https://github.com/graphql/graphql-spec/blob/main/spec/Section%202%20--%20Language.md#inline-fragments ) with no TypeCondition nor Directive.  
**To Reproduce**
Test case on SubscriptionUniqueRootFieldTest

```

 def "subscription with multiple root fields within inline fragment are not allowed"() {
        given:
        def subscriptionOneRootWithFragment = '''
            subscription doggo {             
                ... {
                    dog {
                        name
                    }
                    cat {
                        name
                    }
                }
            }
        '''

        when:
        def validationErrors = validate(subscriptionOneRootWithFragment)

        then:
        !validationErrors.empty
        validationErrors.size() == 1
        validationErrors[0].validationErrorType == ValidationErrorType.SubscriptionMultipleRootFields
        validationErrors[0].message == "Validation error (SubscriptionMultipleRootFields) : Subscription operation 'doggo' must have exactly one root field"
    }
```

same for multiple inline fragments:

```
        def subscriptionOneRootWithFragment = '''
            subscription doggo {             
                ... {
                    ...{
                       dog {
                           name
                       }
                       cat {
                           name
                       }
                   }
                }
            }
        '''
```

**Comment by dondonz** [comment]
I'm reserving this issue 😄

**Comment by Juliano-Prado** [comment]
After talking with @dondonz I have raised a fix: https://github.com/graphql-java/graphql-java/pull/3481

**Comment by dondonz** [comment]
Closed by #3481
